config/locales/views.de.yml

```diff
@@ -747,6 +747,15 @@ de:
         email_taken: E-Mail wird bereits verwendet. Melde dich an oder registrier dich mit einer anderen E-Mail-Adresse.
         flash:
           privacy_policy_not_accepted: Um die Registrierung abzuschliessen, muss der Datenschutzerklärung zugestimmt werden.
+    self_inscription:
+      new:
+        role_exists: Du kannst nicht zweimal die gleiche Rolle in einer Gruppe haben
```

Comment:
Dieser Hinweis stimmt so nicht. Aber vielleicht können wir es so formulieren, dass wir immer noch verhindern können, dass man sich zwei Mal für dieselbe Rolle einschreibt: "Du hast diese Rolle in dieser Gruppe bereits."

Reply:
Ich habe mal deine Version genommen.